crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -10,7 +10,11 @@ All notable changes to this project will be documented in this file.
 
 - [**breaking**] Move `session_id` from `EncryptionInfo` to `AlgorithmInfo` as it is megolm specific. 
   Use `EncryptionInfo::session_id()` helper for quick access.
-  ([#4981](https://github.com/matrix-org/matrix-rust-sdk/pull/4775), [#4864](https://github.com/matrix-org/matrix-rust-sdk/pull/4981))
+  ([4981](https://github.com/matrix-org/matrix-rust-sdk/pull/4981))
+
+- [**breaking**] The `ProcessedToDeviceEvent::Decrypted` variant now also have an `EncryptionInfo` field.
+  Format changed from `Decrypted(Raw<AnyToDeviceEvent>)` to `Decrypted { raw: Raw<AnyToDeviceEvent>, encryption_info: EncryptionInfo) }`
+  ([5074](https://github.com/matrix-org/matrix-rust-sdk/pull/5074))
```

Comment:
Let's move this to the top, we should add new entries at the top. Also did you intentionally remove the second PR link from the previous entry?

Reply:
Ah, ok.